docs/team/becoming-member.md

```diff
@@ -55,8 +61,54 @@ This process takes the following steps:
    if they are interested. Don't forget to run them by the {ref}`membership_guidelines`
    page to make sure they understand what they're signing up for.
 
-## Membership Maintenance
+### Membership Maintenance
 
-Every six months, one currently active member should open an issue in the team-compass repo asking all currently active team members to reply if they still consider themselves active. If not (or no response is given by a team member), it will be assumed that they have gone inactive. This will help keep the active member list up-to-date.
+Every six months, a bot will open an issue in the team-compass repo asking all currently active team members to reply within three weeks if they still consider themselves active. If a team member replies no, or does not reply, we will conclude that they are inactive. This will help keep the active member list up to date.
 
 Remember, an inactive member can return at any time by simply changing their status on the team-compass page.
+
+## Release team member
+
+### Joining the team
+
+Any JupyterLab Council member can ask to be part of the release team using the weekly call, chat, or official council email list.
+
+### Release team responsibilities
+
+Release team members can manage the following teams:
+
+- GitHub JupyterLab organization: membership of the [release team](https://github.com/orgs/jupyterlab/teams/release)
+- PyPI Jupyter organization: manager of the [JupyterLab team](https://pypi.org/manage/organization/jupyter/team/jupyterlab)
+- NPM jupyterlab organization: membership of the [jupyterlab team](https://www.npmjs.com/settings/jupyterlab/teams)
+- conda-forge jupyterlab recipe: maintainer rights
+
+> An admin will need to add a new member to all three teams.
+
+Members of these teams have publication rights for major JupyterLab-related packages. Team members can publish a release and can react quickly in case a published package is broken or corrupted.
+
+### Membership Maintenance
+
+Every six months, a bot will open an issue in the team-compass repo asking all currently team members to reply within three weeks if they still consider themselves active. If a member replies no, or they do not reply, we will conclude that they are inactive. Inactive members may be removed from their teams.
```

Comment:
Because we have similar language for other roles, it would be helpful to be specific about which role the current paragraph applies. This could be addressed by changing "current team members" to "current release team members".

Reply:
Clarify this point.